monai/apps/reconstruction/fastMRIreader.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, Sequence, Tuple, Union
+
+import numpy as np
+from numpy import ndarray
+
+from monai.config import PathLike
+from monai.data.image_reader import ImageReader
+from monai.data.utils import is_supported_format
+from monai.utils import FastMRIKeys, optional_import, require_pkg
+from monai.utils.type_conversion import convert_to_tensor
+
+h5py, has_h5py = optional_import("h5py")
+
+
+@require_pkg(pkg_name="h5py")
+class FastMRIReader(ImageReader):
+    """
+    Load fastMRI files with '.h5' suffix.
+    """
+
+    def __init__(self):
+        super().__init__()
+
+    def verify_suffix(self, filename: Union[Sequence[PathLike], PathLike]) -> bool:
```

Comment:
This func is not used in this PR. Please double check it will be used in somewhere else.

Reply:
Yes, `LoadImage` transform https://github.com/Project-MONAI/MONAI/blob/112d36c659b0cae510f44f78edfddea93ba314d3/monai/transforms/io/array.py#L89 calls ``verify_suffix`.

Every `ImageReader` class https://github.com/Project-MONAI/MONAI/blob/7f80288381575121e670a4e184d45b01813a51be/monai/data/image_reader.py#L66 should define a `verify_suffix`.

Reply:
Great! Thanks!